C/CUDA/CUDA_Runtime/platform_augmentation.jl

```diff
@@ -47,7 +47,7 @@ function cuda_toolkit_tag()
         cuda_driver = if CUDA_Driver_jll.is_available()
             @debug "Using CUDA_Driver_jll for driver discovery"
 
-            if !isdefined(CUDA_Driver_jll, :libcuda)
+            if isnothing(CUDA_Driver_jll.libcuda)
```

Comment:
I'm not sure if this has affected more people, I assume it has, but my environment blew up with this change. I believe this code can be pulled in without the required `CUDA_Driver_jll` changes, which then hits an undefined variable error here.

Reply:
I don't see how that can happen; https://github.com/JuliaBinaryWrappers/CUDA_Runtime_jll.jl/blob/c255c7f1878a00d690d7eb138a656c1d196e38a6/Project.toml#L16

Reply:
Beats me, all I know is it failed, and I don't use any of these packages directly, they're coming in via Flux, so I have no personal experience dealing with it. I fixed it by pinning it to 0.4.0, will look more into what may have happened to the environment state later.